src/components/WalletSuite/ConnectedWallet/Details/MyEndowments/MyEndowments.tsx

```diff
@@ -10,7 +10,7 @@ export default function MyEndowments({ endowments }: Props) {
       <h3 className="font-heading font-bold text-sm text-gray-d1 dark:text-gray">
         My Endowments
       </h3>
-      <div className="overflow-y-auto max-h-40 scroller grid gap-3">
+      <div className="overflow-y-auto max-h-40 grid gap-3">
```

Comment:
```suggestion
      <div className="overflow-y-auto max-h-40 scroller grid gap-3">
```
@satishkumarsajjan, nice work🥇 
we should not remove `scroller` as it is needed when the list gets long
* #1968